blaze/expr/expressions.py

```diff
@@ -845,6 +945,7 @@ def ndim(expr):
 schema_method_list.extend([
     (isscalar, set([label, relabel, coerce])),
     (isrecord, set([relabel])),
+    (lambda ds: isinstance(ds, Option), {coalesce}),
```

Comment:
`coalesce` is valid for any `isscalar` dshape, right? Not just `Option` types?  Or am I misunderstanding coalesce?